crates/bevy_core/src/time/time.rs

```diff
@@ -34,58 +49,171 @@ impl Time {
+        self.fixed_delta_seconds
+    }
+
+    /// The *virtual* delta between each fixed update step, as [`f64`] seconds.
+    /// Note that the true delta between fixed update steps varies.
+    #[inline]
+    pub fn fixed_delta_seconds_f64(&self) -> f64 {
+        self.fixed_delta_seconds_f64
+    }
+
+    /// Changes [`Time::fixed_delta`]. Users are strongly cautioned against modifying
+    /// the step rate this way, except during startup.
+    /// Prefer [`Time::set_relative_speed`] to ensure that consumers of
+    /// [`Time::fixed_delta`] produce consistent behavior.
+    pub fn set_steps_per_second(&mut self, steps: f32) {
+        self.fixed_delta = Duration::from_secs_f32(1.0 / steps);
+        self.fixed_delta_seconds = self.fixed_delta.as_secs_f32();
+        self.fixed_delta_seconds_f64 = self.fixed_delta.as_secs_f64();
+    }
+
+    /// The rate that time advances relative to the real world, as [`f32`].
+    #[inline]
+    pub fn relative_speed(&self) -> f32 {
+        self.relative_speed
+    }
+
+    /// The rate that time advances relative to the real world, as [`f64`].
+    #[inline]
+    pub fn relative_speed_f64(&self) -> f64 {
+        self.relative_speed as f64
+    }
+
+    /// Set rate time advances relative to the real world. Must be >= 0.0.
+    #[inline]
+    pub fn set_relative_speed(&mut self, relative_speed: f32) {
+        assert!(relative_speed >= 0.0);
+        self.relative_speed = relative_speed;
     }
 
-    /// The ['Instant'] when [`Time::update`] was last called, if it exists
+    /// The ['Instant'] when [`Time::update`] was first called, if it exists.
+    #[inline]
+    pub fn first_update(&self) -> Option<Instant> {
+        self.first_update
+    }
+
+    /// The ['Instant'] when [`Time::update`] was last called, if it exists.
     #[inline]
     pub fn last_update(&self) -> Option<Instant> {
         self.last_update
     }
 
+    /// The [`Instant`] the app was started.
+    #[inline]
+    pub fn startup(&self) -> Instant {
+        self.startup
+    }
+
+    /// The time since startup, as [`Duration`]. Affected by speed multiplier.
+    /// Sampled during the latest update.
+    #[inline]
     pub fn time_since_startup(&self) -> Duration {
+        self.time_since_startup
+    }
+
+    /// The time since startup, as [`f32`] seconds. Affected by speed multiplier.
+    /// Sampled during the latest update.
+    #[inline]
+    pub fn seconds_since_startup(&self) -> f32 {
+        self.time_since_startup().as_secs_f32()
+    }
+
+    /// The time since startup, as [`f64`] seconds. Affected by speed multiplier.
+    /// Sampled during the latest update.
+    #[inline]
+    pub fn seconds_since_startup_f64(&self) -> f64 {
+        self.time_since_startup().as_secs_f64()
+    }
+
+    /// The time since startup, as [`Duration`]. Unaffected by speed multiplier.
+    /// Sampled during the latest update.
+    pub fn real_time_since_startup(&self) -> Duration {
```

Comment:
"real" -> "unscaled"?

Reply:
idk, I personally don't like "scale" and "(un)scaled" as replacements for "relative_speed" and "real." If I had to change it, I think "dilation" and "(un)dilated" would be more intuitive for beginners.

Reply:
Un/scaled is what Unity uses. It's not perfect but has the merit of being familiar to many devs.

Reply:
I know. It's a minor nitpick. While I drew a lot from Unity, not everyone coming to Bevy will be jumping from Unity (or Godot). I'm personally more concerned about complete beginners. 

I don't like "scale" referring to a speed, "time scale" in most outside contexts does not mean time dilation, and `Time.deltaTime == Time.timeScale * Time.unscaledDeltaTime`, but `Time.time != Time.timeScale * Time.unscaledTime`.